pom.xml

```diff
@@ -387,11 +387,17 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
```

Comment:
Why you adding this here?
It actually tries to run tests on the parent project, with obviously missing test folder. This is also probably the reason for your line 612. Removing this should make the line 613 pointless.

Reply:
Plugin removed in commit ce7c3fa686ae91c964dc13a408e608ae1c5f17ee.